src/Http/Requests/Account/LoginTwoFaUserCodeRequest.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Blumilk\Meetup\Core\Http\Requests\Account;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class LoginTwoFaUserCodeRequest extends FormRequest
+{
+    public function rules(): array
+    {
+        return [
+            "email" => ["required", "string", "email", "max:255"],
+            "code" => ["required", "string", "numeric", "max:999999"],
```

Comment:
Are we really handling properly 999999-length codes?

Reply:
it is the maximum number so that the user cannot specify more than 6 digits